.circleci/conditional_config.yml

```diff
@@ -39,23 +39,42 @@ images:
 
   - &backend registry.opensuse.org/obs/server/unstable/containers/containers/openbuildservice/backend:latest
 
-  - &frontend_backend
+  - &frontend_minitest
     image: registry.opensuse.org/obs/server/unstable/containers/containers/openbuildservice/frontend-backend:latest
     <<: *common_frontend_config
     environment:
       EAGER_LOAD: 1
 
   - &frontend_base
-    image: registry.opensuse.org/obs/server/unstable/containers/containers/openbuildservice/frontend-base:latest
+    image: registry.opensuse.org/obs/server/unstable/containers/containers/openbuildservice/frontend-base-ci:latest
     <<: *common_frontend_config
 
-  - &frontend_base_ci
-    image: registry.opensuse.org/obs/server/unstable/containers/containers/openbuildservice/frontend-base-ci:latest
+  - &frontend_features
+    image: registry.opensuse.org/obs/server/unstable/containers/containers/openbuildservice/frontend-base:latest
 aliases:
+  - &restore_bundler_cache
+    restore_cache:
+      name: Restoring Bundle Cache
+      key: bundle-{{ checksum "src/api/Gemfile.lock" }}
+
   - &install_dependencies
     name: install dependencies
     command: |
-      cd ./src/api && bundle install --jobs=4 --retry=3
+      cd ./src/api
+      bundle config build.ffi --enable-system-libffi
+      bundle config build.nokogiri --use-system-libraries
+      bundle config build.sassc --disable-march-tune-native
```

Comment:
For ffi and nokogiri, I understand that we rely on the system libraries. For sassc and nio4r though, what are those options for? Could we have a comment to explain what they are?

Reply:
Those are compiler options. They come from [obs-bundled-gems.spec](https://github.com/openSUSE/open-build-service/blame/master/dist/obs-bundled-gems.spec#L125)...

Reply:
Pushed a comment....

Reply:
Is the new TODO going to be addressed in this PR?

Reply:
huh?

Reply:
this is a general TODO. always. just a reminder. nothing you can finish.